src/db.js

```diff
@@ -8,7 +8,7 @@ import {TermModel, TermRelationshipModel, TermTaxonomyModel} from './modules/Ter
 
 // CONNECTORS
 import PostConnectors from './modules/Post/connectors'
-import PostmetaConnectors from './modules/Postmeta/connectors'
+import PostmetaConnectors from './modules/PostMeta/connectors'
```

Comment:
I think for this we should just rename the directory to `Postmeta` to be consistent.